xkcd_dl/annotated.py

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env python3
```

Comment:
Please revert the changes made on `annoted.py`. As we have discussed, it is for reference purposes for other contributors.